motion/motion.go

```diff
@@ -223,22 +221,24 @@ func (d *motionDetector) warmerDiffFrames(a, b, out *lepton3.Frame) *lepton3.Fra
 }
 
 func (d *motionDetector) updateBackground(new_frame *lepton3.Frame) (float64, bool) {
-	if d.count == 0 {
+	d.backgroundFrames++
+	if d.backgroundFrames == 1 {
 		d.background = new_frame.Pix
 		return 0, true
 	}
+	count := float64((d.rowStop - d.start) * (d.columnStop - d.start))
```

Comment:
The name `count` is a bit misleading here as it implies an incrementing value. How about `numPixels` or similar?

Also, this could be precalculated instead of recalculating it every frame.